### PR TITLE
Deleting `WorkflowPluginTest.grapeLibrary`

### PR DESCRIPTION
**Pull request by jglick**
Deleting test added in #176. Known to be flaky.

https://github.com/jenkinsci/pipeline-groovy-lib-plugin/blob/master/src/test/java/org/jenkinsci/plugins/workflow/cps/global/GrapeTest.java covers most of this, and could be extended to use `RealJenkinsRule` for added strength as needed.

### CODE REVIEW

File: src/test/java/plugins/WorkflowPluginTest.java [hunk 1]
```diff
@@ -314,7 +292,7 @@ private String createGitRepositoryInDockerContainer() throws IOException {
         assertTrue(b2.getChanges().hasChanges());
     }
 
-    @WithPlugins({"git@3.0.1", "workflow-job", "workflow-cps", "workflow-basic-steps", "workflow-durable-task-step", "workflow-multibranch", "github-branch-source@2.5.5", "workflow-cps-global-lib"})
+    @WithPlugins({"git@3.0.1", "workflow-job", "workflow-cps", "workflow-basic-steps", "workflow-durable-task-step", "workflow-multibranch", "github-branch-source@2.5.5", "pipeline-groovy-lib"})
```
Comment (jglick) [inline src/test/java/plugins/WorkflowPluginTest.java:295]: see https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/172

File: src/test/java/plugins/WorkflowPluginTest.java [hunk 2]
```diff
@@ -279,24 +275,6 @@ private String createGitRepositoryInDockerContainer() throws IOException {
         }
     }
 
-    @WithPlugins({"workflow-cps-global-lib@2.3", "workflow-basic-steps@2.1", "workflow-job@2.5"})
-    @Issue("JENKINS-26192")
-    @Test public void grapeLibrary() throws Exception {
-        assumeThat("TODO otherwise we would need to set up SSH access to push via Git, which seems an awful hassle", controller, instanceOf(LocalController.class));
-        File workflowLibs = /* WorkflowLibRepository.workspace() */ new File(((LocalController) controller).getJenkinsHome(), "workflow-libs");
```
Comment (jglick) [inline src/test/java/plugins/WorkflowPluginTest.java:286]: Also was using long-since-deprecated `workflowLibs.git` rather than an SCM-controlled lib.

**Review by jtnord — APPROVED** [review]

**Comment by jglick** [comment]
@timja I guess the test failures are unrelated?

**Comment by NotMyFault** [comment]
> @timja I guess the test failures are unrelated?

See https://github.com/jenkinsci/acceptance-test-harness/pull/981, I've had no shot to investigate yet.

**Comment by timja** [comment]
https://github.com/jenkinsci/acceptance-test-harness/issues/985

**Pull request merged**